core/src/avm2/globals/xml.rs

```diff
@@ -128,15 +129,60 @@ pub fn set_name<'gc>(
     Ok(Value::Undefined)
 }
 
-pub fn namespace<'gc>(
+pub fn _namespace<'gc>(
```

Comment:
Non-blocking, but this name makes it sound like the function is unused. Maybe `internal_get_namespace` (or something similar)? Alternatively, we could keep it entirely implemented in Rust, and base the behavior off of `args.len()`.